curriculum/challenges/english/25-front-end-development/lab-movie-review-page/67ef8f695db52583424bd118.md

```diff
@@ -67,18 +63,24 @@ assert.isAtLeast(paragraphs.length, 2);
 Inside the second `p` element, you should have a `strong` element with the text `Movie Rating`.
 
 ```js
-const strongEl = document.querySelector("main p strong");
+onst strongEl = document.querySelector("main p:nth-of-type(2) strong");
 assert.match(strongEl?.innerText.trim(), /^Movie Rating:?$/);
 ```
 
-There should be a `span` element inside the rating paragraph with `aria-hidden` set to `true`.
+There should be a `span` element inside the rating paragraph.
```

Comment:
You are not saying that you want some text inside the `span`. Also, how many stars they are supposed to use?

Reply:
The learner is open to use any number os stars, there is no specific set number